plantcv/plantcv/visualize/display_instances.py

```diff
@@ -0,0 +1,200 @@
+# display instances in an image
+import cv2
+from matplotlib.patches import Polygon
+import colorsys
+import random
+from skimage.measure import find_contours
+import matplotlib.pyplot as plt
+import numpy as np
+from matplotlib import patches, lines
+from matplotlib.patches import Polygon
+# from plantcv.plantcv.visualize import overlay_two_imgs
+# from plantcv.plantcv.visualize import colorize_masks
+from plantcv import plantcv as pcv
+from plantcv.plantcv import fatal_error
+
+def _overlay_mask_on_image(image, mask, color, alpha=0.5):
+    """ apply a mask to an input image with a user defined color
+    :param image: input RGB or grayscale image
+    :param mask: desired mask
+    :param color: (a tuple) desired color to show the mask on top of the image
+    :param alpha: (a value between 0 and 1) transparency value when blending mask and image, by default 0.5
+    :return: image with an mask with desired color on top of it
+    """
+    if len(image.shape) == 2:
+        image = cv2.cvtColor(image, cv2.COLOR_GRAY2BGR)
```

Comment:
Looks like this line isn't covered by tests currently. We might need to create a grayscale img input for one of the tests to plot on?

Reply:
You are right. 
I will update the tests